lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -26,18 +26,26 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
    * @type {String[]}
    * @public
    */
-  this.mainSupportedExtensions = _.slice(SupportedFormatsModel.getAllExtensions(), 0, 3);
+  this.mainSupportedExtensions = _.intersection([
+    'img',
+    'iso',
+    'zip'
+  ], SupportedFormatsModel.getAllExtensions());
 
   /**
    * @summary Extra supported extensions
    * @constant
    * @type {String[]}
    * @public
    */
-  this.extraSupportedExtensions = _.difference(
+  this.extraSupportedExtensions = _.sortBy(_.difference(
     SupportedFormatsModel.getAllExtensions(),
     this.mainSupportedExtensions
-  );
+
+  // Sort alphabetically
+  ), (extension) => {
+    return extension.charCodeAt();
```

Comment:
Oh, I completely ignored the fact `.sort()` worked out of the box for strings. I'll fix!